server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/node/metedata/read/SchemaScanNode.java

```diff
@@ -116,13 +123,31 @@ public void setHasLimit(boolean hasLimit) {
     this.hasLimit = hasLimit;
   }
 
-  public abstract List<String> getOutputColumnNames();
+  @Override
+  public <R, C> R accept(PlanVisitor<R, C> visitor, C context) {
```

Comment:
abtract class don't need to override this method.

Reply:
removed